image/ubuntu-image/scripts/install_dependencies.sh

```diff
@@ -54,16 +54,16 @@ if [ "$PKGSTOINSTALL" != "" ]; then
     fi
 
     # upgrade
-    apt-get -y upgrade | tee /tmp/bibos_upgrade_log.txt
-    apt-get -y dist-upgrade | tee /tmp/bibos_upgrade_log.txt
+    apt-get -y upgrade | tee /tmp/os2borgerpc_upgrade_log.txt
+    apt-get -y dist-upgrade | tee /tmp/os2borgerpc_upgrade_log.txt
 
     # Clean .deb cache to save space
     apt-get -y autoremove
     apt-get -y clean
 fi
 
-# Install python packages
-pip install --upgrade bibos-utils bibos-client
+# Install os2borgerpc client
+pip3 install http://bibos-admin.magenta-aps.dk/archive/client_packages/os2borgerpc_client-0.0.5.1.tar.gz
```

Comment:
Hm. This _feels_ like a temporary hack to work around the fact that we haven't made a release for a while. :-)

Should it really be merged?

Reply:
I think it should be merged but not released. We're not quite ready to release the Python 3 client yet. As soon as we are, we can fix this on a separate PR.

Reply:
Fixed.